ddsrecorder/test/blackbox/both/ResourceLimitsTest.cpp

```diff
@@ -242,7 +242,7 @@ class ResourceLimitsTest : public testing::Test
         const auto file_size = std::filesystem::file_size(file_path);
         const auto is_acceptable =
                 file_size >= limits_->MIN_ACCEPTABLE_FILE_SIZE / (mcap_file ? 2.1 : 1) &&
-                file_size <= limits_->MAX_ACCEPTABLE_FILE_SIZE / (mcap_file ? 1.9 : 1);
+                file_size <= limits_->MAX_ACCEPTABLE_FILE_SIZE;// / (mcap_file ? 1.9 : 1);
```

Comment:
I guess this should be removed?
```suggestion
                file_size <= limits_->MAX_ACCEPTABLE_FILE_SIZE;
```

Reply:
Done in 55d0ea7a97a1b1168825b06079dfb1c447d1cf07